uaclient/cli/cves.py

```diff
@@ -0,0 +1,143 @@
+import sys
+
+from uaclient import exceptions, messages
+from uaclient.api.u.pro.security.cves.v1 import (
+    CVEVulnerabilitiesOptions,
+    _vulnerabilities,
+)
+from uaclient.cli import cli_util
+from uaclient.cli.commands import ProArgument, ProArgumentGroup, ProCommand
+from uaclient.cli.formatter import Table, create_link
+from uaclient.config import UAConfig
+
+
+@cli_util.with_spinner(msg=messages.CLI_CVES_SPINNER_MSG)
+def _get_cve_vulnerabilities(args, *, cfg: UAConfig, **kwargs):
+    cve_options = CVEVulnerabilitiesOptions(
+        fixable=args.fixable,
+        unfixable=args.unfixable,
+    )
+    return _vulnerabilities(options=cve_options, cfg=cfg)
+
+
+def _colorize_priority(priority):
+    if priority == "low":
+        return messages.TxtColor.INFOBLUE + priority + messages.TxtColor.ENDC
+    elif priority == "medium":
+        return (
+            messages.TxtColor.WARNINGYELLOW + priority + messages.TxtColor.ENDC
+        )
+    elif priority == "high":
+        return messages.TxtColor.ORANGE + priority + messages.TxtColor.ENDC
+    elif priority == "critical":
+        return messages.TxtColor.FAIL + priority + messages.TxtColor.ENDC
+    else:
+        return priority
+
+
+def _get_cve_table_rows(cve_vulnerabilities):
+    rows = []
+
+    for package_name, package_info in cve_vulnerabilities.packages.items():
+        for cve in package_info.cves:
+            cve_info = cve_vulnerabilities.cves.get(cve.name)
+
+            if cve_info:
+                rows.append(
+                    (
+                        package_name,
+                        cve_info.priority,
+                        cve.fix_origin or "-",
+                        create_link(
+                            text=cve.name,
+                            url="https://ubuntu.com/security/{}".format(
+                                cve.name
+                            ),
+                        ),
```

Comment:
Nit. `create_link` arguably is formatting for these values similar to the coloring of the priority. Maybe the `create_link` call belongs in `_format_cve_rows` instead of here?